tsconfig-aot.json

```diff
@@ -13,7 +13,12 @@
     "noEmitHelpers": true,
     "rootDir": ".",
     "noUnusedLocals": true,
-    "noUnusedParameters": true
+    "noUnusedParameters": true,
+    "types": [
```

Comment:
This array isn't actually needed. Any global definitions that are `@types` scoped will be automatically included if this entry isn't there